src/backends/cgpm_backend.py

```diff
@@ -922,6 +923,76 @@ def logpdf_joint(
             multiprocess=self._multiprocess,
         )
 
+    # Return python data object that's ready to be written in JSON syntax
+    def json_ready_models(self, bdb, population_name):
```

Comment:
Please add `json_ready_models` as method in `BayesDB_Backend` interface, defined in `src/backend.py`